local/local_test.go

```diff
@@ -1266,8 +1403,8 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 			inspect, _, err := dockerClient.ImageInspectWithRaw(context.TODO(), prevName)
 			h.AssertNil(t, err)
 
-			prevLayer1SHA = inspect.RootFS.Layers[0]
-			prevLayer2SHA = inspect.RootFS.Layers[1]
+			prevLayer1SHA = inspect.RootFS.Layers[len(inspect.RootFS.Layers)-2]
+			prevLayer2SHA = inspect.RootFS.Layers[len(inspect.RootFS.Layers)-1]
```

Comment:
Is this to avoid having the assertion switch on the daemon OS?

Reply:
Yes, and I realize it's repeated in a couple other places - I'll toss in a helper to clarify it. 

The reason it needed to be changed here is that `NewImage()` can no longer create Windows images with no layers at all (which technically *could* be saved to a daemon, but have no value that I could see).

If layerless Windows images are important, I can certainly take a shot at getting them back

Reply:
I [added](https://github.com/buildpacks/imgutil/pull/91/files#diff-fd2fec5210d5a5d51c6442448b9a8577ad13498d133c4d9c4839ce292667edb5R359) a new helper that works as follows:

from:
```golang
prevLayer1SHA = inspect.RootFS.Layers[len(inspect.RootFS.Layers)-2]
```
to:
```golang
// negative index (offset from len(inspect.RootFS.Layers))
prevLayer1SHA = h.StringElementAt(inspect.RootFS.Layers), -2)

// also normal index
prevLayer1SHA = h.StringElementAt(inspect.RootFS.Layers), 1)
```

Reply:
> The reason it needed to be changed here is that NewImage() can no longer create Windows images with no layers at all (which technically could be saved to a daemon, but have no value that I could see).

This statement confuses me. I thought we were always adding a base layer on windows images because it is impossible to pull a windows image without one. Can you successfully docker load an empty image, even though you can't pull one?